include/circt/Dialect/FIRRTL/FIRRTLDialect.h

```diff
@@ -22,7 +22,4 @@
 // Pull in the dialect definition.
 #include "circt/Dialect/FIRRTL/FIRRTLDialect.h.inc"
 
-// Pull in all enum type definitions and utility function declarations.
-#include "circt/Dialect/FIRRTL/FIRRTLEnums.h.inc"
```

Comment:
Was including this a problem?
Looking at upstream MLIR dialect headers, they seem to.. do things differently.

I think requiring including FIRRTLEnums.h separately is reasonable to match how we've organized attributes/operations/types similarly.

Good call, thanks!